src/cli/init.rs

```
@@ -129,17 +141,14 @@ pub async fn execute(args: Args) -> miette::Result<()> {
     Ok(())
 }
 
-// Checks if string is in file.
-// If search string is multiline it will check if any of those lines is in the file.
-fn string_in_file(path: &Path, search: &str) -> bool {
-    let content = fs::read_to_string(path).unwrap_or_default();
-    search.lines().any(|line| content.contains(line))
-}
-
 // When the specific template is not in the file or the file does not exist.
 // Make the file and append the template to the file.
 fn create_or_append_file(path: &Path, template: &str) -> miette::Result<()> {
-    if !path.is_file() || !string_in_file(path, template) {
+    let file = fs::read_to_string(path)
+        .into_diagnostic()
```

no need to turn this into a diagnostic first if you ignore the error anyway.